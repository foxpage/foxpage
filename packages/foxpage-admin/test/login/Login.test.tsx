import React from 'react';
import { Provider } from 'react-redux';

import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';

import Login from '../../src/pages/login';

const mockStore = configureStore([]);

describe('@foxpage/foxpage-admin', () => {
  it('match snapshot', async () => {
    const wrapper = shallow(
      <Provider
        store={mockStore({
          login: {
            loading: true,
          },
        })}
      >
        <Login />
      </Provider>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
