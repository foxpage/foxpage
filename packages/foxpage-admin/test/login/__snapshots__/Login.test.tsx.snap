// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@foxpage/foxpage-admin match snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "onStateChange": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <Connect(Login) />
</ContextProvider>
`;
